src/_micro-api/libraries/signalDecoder/src/signalDecoder.cpp

```diff
@@ -589,7 +509,7 @@ void SignalDetectorClass::processMessage()
 #endif
 		}
 	}
-	if (!m_truncated)
+	if (!m_truncated && !bufferModified) 
```

Comment:
Zu dem Problem (ich glaube es ist das gleiche) habe ich gestern eine Änderung commited.
ich habe mc_sync auf false gesetzt, sobald ein nicht konformes Signa gefunden wird.

Bin aber noch nicht sicher, ob das in allen Fällen gut ist.